resources/vocabulary.md

```diff
@@ -93,8 +93,18 @@ sum_of_nums_under(4) // 10
 sum_of_nums_under(6) // 21
 ```
 
+## Process and Tools
```

Comment:
I think we decided to split into 2 files as part of the re-org, so I'll let this be for the moment